doc/sphinx/language/core/assumptions.rst

```diff
@@ -110,6 +110,7 @@ Function application
    term_application ::= @term1 {+ @arg }
    | @ @qualid_annotated {+ @term1 }
    arg ::= ( @ident := @term )
+   | ( @natural := @term )
```

Comment:
Would you add phrase or sentence explaining the new syntax?  Probably in the paragraph at line 123 ("The notation ...").

Reply:
I made a try.

Reply:
Looks good.